src/go/rdctl/pkg/client/utils.go

```diff
@@ -56,14 +56,14 @@ func ProcessRequestForUtility(response *http.Response, err error) ([]byte, error
 	if response.StatusCode < 200 || response.StatusCode >= 300 {
 		// Note that response.Status includes response.StatusCode
 		switch response.StatusCode {
-		case 400:
+		case http.StatusBadRequest:
 			statusMessage = response.Status
 			// Prefer the error message in the body written by the command-server, not the one from the http server.
-		case 401:
+		case http.StatusUnauthorized:
 			return nil, fmt.Errorf("%s: user/password not accepted", response.Status)
-		case 413:
+		case http.StatusRequestEntityTooLarge:
```

Comment:
I think I would find it useful to keep the numeric status codes in a comment, e.g.

```go
case http.StatusRequestEntityTooLarge: // 413
```

I partially negates the advantage of using named constants, but it helps when you are looking at error messages and can find the actual status code in the source.

Reply:
I think the more important part of using the constants is to prevent typos (because a typo on a number is just a different valid number); including the actual number as a comment is still good in that it makes debugging easier, though.